save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/Warning.kt

```diff
@@ -33,19 +33,22 @@ data class Warning(
  * @param messageGroupIdx index of capture group for waring text
  * @param fileName file name
  * @param line line number of warning
+ * @param benchmarkMode whether to ignore the warning messages
  * @return a [Warning] or null if [this] string doesn't match [warningRegex]
  * @throws ResourceFormatException when parsing a file
  */
+@Suppress("TOO_MANY_PARAMETERS", "LongParameterList")
 internal fun String.extractWarning(warningRegex: Regex,
                                    fileName: String,
                                    line: Int?,
                                    columnGroupIdx: Long?,
                                    messageGroupIdx: Long,
+                                   benchmarkMode: Boolean,
 ): Warning? {
     val groups = warningRegex.find(this)?.groups ?: return null
 
     val column = getRegexGroupSafe(columnGroupIdx, groups, this, "column number")?.toIntOrNull()
-    val message = getRegexGroupSafe(messageGroupIdx, groups, this, "warning message")!!.trim()
+    val message = if (!benchmarkMode) getRegexGroupSafe(messageGroupIdx, groups, this, "warning message")!!.trim() else " "
```

Comment:
Why `" "` and not `""`?

Reply:
We have several unchecked cases in `StringUtils`, e.g. we don't consider situations, when we've got an empty strings and it leads to the exceptions

This PR was aimed to be as fast solution, so I didn't fixed `StringUtils` here and used this workaround